app/src/main/kotlin/com/vestrel00/daggerbutterknifemvp/ui/common/presenter/Presenter.kt

```diff
@@ -14,48 +14,47 @@
  * limitations under the License.
  */
 
-package com.vestrel00.daggerbutterknifemvp.ui.common.presenter;
+package com.vestrel00.daggerbutterknifemvp.ui.common.presenter
 
-import android.os.Bundle;
-import android.support.annotation.Nullable;
+import android.os.Bundle
 
 /**
  * A presenter that defines its own lifecycle methods.
  */
-public interface Presenter {
+interface Presenter {
 
     /**
      * Starts the presentation. This should be called in the view's (Activity or Fragment)
      * onCreate() or onViewStatedRestored() method respectively.
      *
      * @param savedInstanceState the saved instance state that contains state saved in
-     *                           {@link #onSaveInstanceState(Bundle)}
+     *  [onSaveInstanceState].
      */
-    void onStart(@Nullable Bundle savedInstanceState);
+    fun onStart(savedInstanceState: Bundle?) = Unit
```

Comment:
Explicitly declaring the return type of this interface method (`= Unit`) makes it optional for implementors to override.